### PR TITLE
Instalation part added

### PR DESCRIPTION
**Pull request by yarigpopov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,21 @@ This library currently includes the following charts:
 
 Note: This library is still in beta, please use carefully, and file issues as discovered. Pull requests for additional charts always welcome!
 
+## Install
+First install `ember-sparkes` to your application:
```
Comment (ghost) [inline README.md]: spelling: `ember-sparkles`

File: README.md [hunk 2]
```diff
@@ -17,6 +17,21 @@ This library currently includes the following charts:
...
+```bash
+ember install ember-sparkes
```
Comment (ghost) [inline README.md]: spelling: `ember install ember-sparkles`

File: README.md [hunk 3]
```diff
@@ -17,6 +17,21 @@ This library currently includes the following charts:
...
+```
+Then add default `ember-resize` configuration into `config/environment.js` file:
```
Comment (ghost) [inline README.md]: Then add default `ember-resize` configuration into `config/environment.js` file, these properties can be changed to suit your needs. For more information, please see the documentation for [ember-resize](https://github.com/mike-north/ember-resize):

**Review by ghost — CHANGES_REQUESTED** [review]
thanks very much for submitting this! added some comments for spelling and wording.

**Comment by yarigpopov** [comment]
committed changes

**Review by ghost — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**